src/fmu/ensemble/ensemble.py

```diff
@@ -205,6 +210,20 @@ def _shortcut2path(keys, shortpath):
         # calling function handle further errors.
         return shortpath
 
+    def InitScratchRealization(self, realdir):
```

Comment:
Not pretty that we had to wrap the initializier here. Should probably not be camel-case if so. And not all arguments are included here?

Reply:
Working on it.... in the next commit :)